benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -523,72 +539,102 @@ int main(int argc, char *argv[])
   for (auto const &spec : specs)
   {
 #ifdef KOKKOS_ENABLE_SERIAL
+    ArborX::BVH<Kokkos::Serial::device_type> tmp_bvh;
+    std::cout << "size = " << tmp_bvh.size() << std::endl;
```

Comment:
Somehow this does not produce a "deprecated" warning.

Reply:
Partial specialization deprecation did not work. Opted for deprecating constructors instead.